script_text.py

```diff
@@ -35,7 +35,7 @@
     for center in r.json()['centers']:
         for session in center['sessions']:
             if session['available_capacity'] > 0 \
-                and session['min_age_limit'] == age:
+                and session['min_age_limit'] <= age:

```

Comment:
The statement returns true for someone who is 45+ and the health center has min_age_limit as 18. It should return false because they can only vaccinate at centers with 45+ age limit